SafeExamBrowser.UserInterface.Mobile/Windows/Taskbar.xaml.cs

```diff
@@ -159,5 +161,18 @@ private void InitializeTaskbar()
 			Loaded += (o, args) => InitializeBounds();
 			QuitButton.Clicked += QuitButton_Clicked;
 		}
+
+		void ITaskbar.Focus(bool fromTop)
```

Comment:
The desktop taskbar has substantially more changes, I don't know if they had been forgotten here or if I'm just not seeing them on GitHub?